GUI/Main.cs

```diff
@@ -21,10 +21,11 @@ public enum GUIModFilter
         Compatible = 0,
         Installed = 1,
         InstalledUpdateAvailable = 2,
-        NewInRepository = 3,
-        NotInstalled = 4,
-        Incompatible = 5,
-        All = 6
+        Cached = 3,
```

Comment:
I actually forgot we save The current filter (although i wrote The code for it). Will change it.

----- Ursprüngliche Nachricht -----
Von: "Paul Fenwick" <notifications@github.com>
Gesendet: ‎01.‎09.‎2015 09:32
An: "KSP-CKAN/CKAN" <CKAN@noreply.github.com>
Cc: "Martin Panzer" <postremus1996@googlemail.com>
Betreff: Re: [CKAN] Added a cached filter (#1426)

In GUI/Main.cs:
> @@ -21,10 +21,11 @@ public enum GUIModFilter
>          Compatible = 0,
>          Installed = 1,
>          InstalledUpdateAvailable = 2,
> -        NewInRepository = 3,
> -        NotInstalled = 4,
> -        Incompatible = 5,
> -        All = 6
> +        Cached = 3,
I know that we save our filter in as ActiveFilter in GUIConfig.xml, which meas that re-ordering these means that users may mysteriously have their default filter change if they're coming from an older version.
Is there any chance we can add Cached to the end of the enum rather than in the middle?
—
Reply to this email directly or view it on GitHub.